roles/kafka_controller/templates/client.properties.j2

```diff
@@ -0,0 +1,8 @@
+# Maintained by Ansible
+# Note: Secrets file for decryption when secrets protection is enabled can be found in {{ ssl_file_dir_final }}/kafka-broker-client-security.properties
+{# kafka_controller_client_properties defined in the variables role #}
+{% for key, value in kafka_controller_client_properties|dictsort%}
+{{key}}={{value}}
+{% endfor %}
+default.api.timeout.ms=20000
+request.timeout.ms=20000
```

Comment:
```suggestion
request.timeout.ms=20000
confluent.use.controller.listener=true
```
might help just to put this here and not add the line each time the health check runs

Reply:
yes, this is a better way. Thanks